sys/net/gnrc/link_layer/netdev2/gnrc_netdev2_eth.c

```diff
@@ -0,0 +1,195 @@
+#include "net/gnrc.h"
+#include "net/gnrc/gnrc_netdev2.h"
+#include "net/ethernet/hdr.h"
+
+#include "od.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+static gnrc_pktsnip_t *_recv(gnrc_netdev2_t *gnrc_netdev2)
```

Comment:
@authmillenon Could you help me rewriting this so the packet doesn't get copied twice?